src/client/src/term/disclaimerFallback.js

```diff
@@ -16,6 +19,9 @@ swisstopo reserves the right to alter or remove the content, in full or in part,
 Liability claims against swisstopo for material or immaterial damage resulting from access to or use or non-use of the published information, from misuse of the connection or from technical faults are excluded.
 
 Further legal provisions can be found here: https://www.swissgeol.ch/datenschutz-en/
+
+# Data Collection
+In order to tailor our online offering optimally to your needs, we use the Google Analytics analysis tools. Your behavior on the website is recorded in an anonymized form. Therefore, no personal data is transmitted or stored. If you do not agree with this, you can prevent data collection by these analysis tools and still use this website without any restrictions. For more information, please see our [Legal Notice](https://www.swissgeol.ch/datenschutz).
```

Comment:
PP: Wie auf Zeile 21 auf die richtige Übersetzung verlinken. Betrifft auch `fr` und `it`